docs/community/devstack.md

```diff
@@ -0,0 +1,139 @@
+
+!!! warning
+    docker-compose 2.x (the golang rewrite) has compatibility issues with our current stack, so please use the 1.x version that was written in python.
+
+### Profiles & Configuration
+
+The galaxy_ng project is a highly configurable application, but only at startup time. Almost all of the behavior of the application is influenced by environment variables and static settings.py files that are read in by [django](https://www.djangoproject.com/) and [dynaconf](https://www.dynaconf.com/). None of these settings live in the database currently, so they can only be declared before the application starts.
+
+The "community" profile of galaxy_ng is a combination of settings driven by the docker-compose files found in [the standalone-community directory](https://github.com/ansible/galaxy_ng/tree/master/dev/standalone-community). To get his profile to run, you must do a few things:
+
+1. Copy the ".compose.env.example" file at the root of the repository to a new file named ".compose.env".
+2. Change the COMPOSE_PROFILE value in ".compose.env" from "standalone" to "standalone-community".
+3. Add SOCIAL_AUTH_GITHUB_KEY=<value> to the bottom of ".compose.env".
+4. Add SOCIAL_AUTH_GITHUB_SECRET=<value> to the bottom of ".compose.env".
+5. Add SOCIAL_AUTH_LOGIN_REDIRECT_URL = "/" to the bottom of galaxy_ng/app/settings.py
+6. Add SOCIAL_AUTH_REDIRECT_IS_HTTPS = True to the bottom of galaxy_ng/app/settings.py if your stack will be behind an http server with SSL enabled.
+
+The github key and secret are provided by configuring an oauth account on github.com under your user's developer settings. The oauth application config needs to have a valid+resolvable homepage url and the callback url should be the same host with "/complete/github/" as the path. For local testing, you can use something like http://localhost:8002 as the host.
+
+Afterwards, the stack can be created and spun up via:
+
+1. make docker/all
+2. ./compose up
+
+
+You can how reach the api at http://localhost:5001. The makefile targets created a couple test users, namely admin:admin for the primary superuser. This is purely an API stack with no UI. That makes it difficult to test and use github users, so see the next section about adding the UI.
+
+
+### React.js UI
+
+To connect to UI, the default address is http://localhost:8002. If social auth is enabled, you'll be forced to auth via github if clicking on the "login" link at the top right of the page. The presence of SOCIAL_AUTH_GITHUB_KEY & SOCIAL_AUTH_GITHUB_SECRET in the backend configuration triggers dynaconf to set a feature flag for external auth that the UI reads and alters the login link accordingly. If you want to bypass github and use a local django user, go directly to the http://localhost:8002/ui/login url and login that way.
```

Comment:
Should it be "To connect to the UI" instead of "To connect to UI" ?

Reply:
Maybe use GitHub instead of github in the context of "You'll be forced to auth via github" ?